index.html

```diff
@@ -1,348 +1,331 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles.css" />
-    <title>Tommymarvel Portfolio</title>
-  </head>
 
-  <body>
-    <div class="firstpage">
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="styles.css" />
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+
+  <title>Tommymarvel Portfolio</title>
+</head>
+
+<body>
+  <div class="firstpage">
     <header class="white">
       <a class="logo" href="#">My logo</a>
-      <nav class="nav-bar">
-        <img id="hambuger" src="images/Union.png" alt="hamburger" />
+      <nav class="nav-bars">
+        <div id="hamburger">
+          <i class="fa fa-bars"></i>
+        </div>
+        <div class="mobile-menu">
+          <a href="#" class="close-icon" id="closemenu"><img src="/images/Icon.png" alt="X icon">
+            ></a>
```

Comment:
- [OPTIONAL] Kindly the close-icon is not showing when the hamburger is pop-up. fix that and let's see your project get approved.
 
Your Design
![Screenshot from 2022-07-20 12-49-00](https://user-images.githubusercontent.com/77842237/179952971-5f5e84a6-84bc-4469-84ac-a54dd7928c75.png)